Sources/Components/IconCollection/IconCollectionView.swift

```diff
@@ -98,3 +132,11 @@ private final class CollectionView: UICollectionView {
         }
     }
 }
+
+private extension Array {
+    func chunked(into size: Int) -> [[Element]] {
+        return stride(from: 0, to: count, by: size).map {
+            Array(self[$0 ..< Swift.min($0 + size, count)])
```

Comment:
Is there some sort of conflicting `min` func in `Array`?

Reply:
I got this error. Couldn't figure out how to get around it, other than to use `Swift`.

> Use of 'min' nearly matches global function 'min' in module 'simd' rather than instance method 'min(by:)'